docs/ADV_STABLE_BASELINES_3.md

```diff
@@ -26,6 +26,22 @@ Main Features
 pip install godot-rl[sb3]
 ```
 
+## Basic Environment Usage
+Usage instructions for envs **BallChase**, **FlyBy** and **JumperHard.**
+
+• Download the env:
+
+```bash
+gdrl.env_from_hub -r edbeeching/godot_rl_<ENV_NAME>
+chmod +x examples/godot_rl_<ENV_NAME>/bin/<ENV_NAME>.x86_64 # linux example
+```
+
+• Train a model from scratch:
+
+```bash
+gdrl --env=gdrl --env_path=examples/godot_rl_<ENV_NAME>/bin/<ENV_NAME>.x86_64 --viz
```

Comment:
@edbeeching I am not sure if `--viz` works with sb3? I know you also have this code snippet linked in the main README.md but for me --viz does not seem to work on sb3. If that's the case I can remove it from the snippets.